scripts/run-apisix-on-docker.sh

```diff
@@ -177,17 +152,24 @@ download_docker_image() {
   debug "downloaded APISIX docker image"
 }
 
+download_cloud_module() {
+    debug "downloading cloud lua module ..."
+
+    /bin/bash <(curl -fsSL 'https://raw.githubusercontent.com/api7/cloud-scripts/a9fa31ae0518e5188f66b42a5e46042b75cad993/scripts/cloud-module-fetcher.sh')
```

Comment:
There are two `download_cloud_module` function ?

Reply:
Where is the other one?